### DISCUSSION TITLE
Remove citation at source.yaml

### DISCUSSION THREAD

**Discussion by ecamacho90** [greenelab/lab-website-template#248] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/ecamacho90/CamachoAguilarLab-Website/tree/main

### Version of Lab Website Template you are using

1.1.6

### Description

Hi!
I am trying to remove a citation by adding `remove: true` in the source.yaml but it keeps appearing on the website. I would really appreciate it if you could help me with that. Thanks in advance!

**Comment by vincerubinetti** [comment, marked as answer]
That feature was only added in v1.2.0. You'll have to update your template. Or, if you look at the latest code and search "remove" in cite.py, you'll see two simple lines of python code that you can copy into your template to allow the feature.
